openwisp_utils/tests.py

```diff
@@ -48,6 +53,8 @@ def display_slow_tests(self):
         for name, elapsed in self.test_timings:
             if elapsed > self.slow_test_threshold[0]:
                 slow_tests_counter += 1
+                # Remove docstring if present
+                name = name.split('\n')[0]
```

Comment:
are you sure that there will be always at least one item after the split? Otherwise this line will break.

Reply:
Yes, the `name` string is supposed to be a **non-empty** string. As long as it's true, this line won't break.
Below is the output of above line with and without docstring.
```python
In [1]: 'test_status_signal_emitted (test_project.tests.test_test_utils.TestUtils)\nTests the catch_signal test utility function'.split('\n')     # with docstring                                                               
Out[1]: 
['test_status_signal_emitted (test_project.tests.test_test_utils.TestUtils)',
 'Tests the catch_signal test utility function']

In [2]: 'test_status_signal_emitted (test_project.tests.test_test_utils.TestUtils)'.split('\n')    # without docstring                                                                                                         
Out[2]: ['test_status_signal_emitted (test_project.tests.test_test_utils.TestUtils)']
```